From e379d43a356396e7001b228cd87681dba07db035 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Tue, 21 May 2024 12:06:07 +0300
Subject: [PATCH] [SW-0] pll: fix bug in printing pll lock failure

The pll lock timeout error is not printed due to
carriage return(\r) in preboot_puts. This patch
fixes this by printing new line after the pll name.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Iae539e243b5199a1a48f9ca2e7e4d2b424e50687
---

diff --git a/lib/common/pll.c b/lib/common/pll.c
index 0370013..6c37b0b 100644
--- a/lib/common/pll.c
+++ b/lib/common/pll.c
@@ -296,6 +296,7 @@
 	if (tout < 0) {
 		preboot_puts_raw("PLL lock timedout for");
 		preboot_puts(cfg->name);
+		preboot_puts("\n");
 		is_locked = false;
 	}
 
